inc/saiport.h

```diff
@@ -586,6 +586,22 @@ typedef enum _sai_port_path_tracing_timestamp_type_t
 
 } sai_port_path_tracing_timestamp_type_t;
 
+/**
+ * @brief Attribute data for #SAI_PORT_ATTR_UNRELIABLE_LOSS_OF_SIGNAL
+ */
+typedef enum _sai_port_unreliable_loss_of_signal_t
+{
+    /**  Unreliable loss of signal is not enabled for transceiver */
+    SAI_PORT_UNRELIABLE_LOSS_OF_SIGNAL_DISABLE,
+
+    /**  Unreliable loss of signal is enabled for transceiver */
+    SAI_PORT_UNRELIABLE_LOSS_OF_SIGNAL_ENABLE,
+
+    /**  Unreliable loss of signal is enabled only for interested transceiver */
+    SAI_PORT_UNRELIABLE_LOSS_OF_SIGNAL_AUTO,
```

Comment:
@JaiOCP The auto config from SONIC will be handled by xcvrd currently based on multiple parameters like the transceiver's interface_type, speed, and current ULOS setting whether the PHY setting needs to be enabled or disabled on this port. Is this additional handling planned to be moved to SAI layer? This requires SAI to handle ULOS auto for every interface type set, speed set and ULOS on/off. Can we define SAI_PORT_ATTR_UNRELIABLE_LOSS_OF_SIGNAL as a boolean instead?